src/Protocol/SymbolInformation.php

```diff
@@ -91,6 +90,14 @@ public static function fromNode(Node $node, string $fqn = null)
         }
         if ($node instanceof Node\Name) {
             $symbol->name = (string)$node;
+        } else if(
+            $node instanceof Node\Expr\FuncCall
+            && $node->name instanceof Node\Name
+            && strtolower((string)$node->name) === 'define'
+            && isset($node->args[0])
+            && $node->args[0]->value instanceof Node\Scalar\String_
+        ) {
+            $symbol->name = (string)$node->args[0]->value->value;
```

Comment:
I need to. Otherwise it would return null in this line: https://github.com/jens1o/php-language-server/blob/7380acc49fbacb4d5a6b3a7bf76d6dd55eed2043/src/Protocol/SymbolInformation.php#L88-L90 

And otherwise it would get resetted when I don't check it twice.